includes/gateways/stripe/includes/admin/class-give-stripe-admin-settings.php

```diff
@@ -676,105 +687,195 @@ public function register_advanced_settings( $settings ) {
 		 *
 		 * @param string $status Status - Enabled or Disabled.
 		 *
+		 * @return string
 		 * @since  2.5.0
 		 * @access public
 		 *
-		 * @return string
 		 */
 		public function stripe_modal_checkout_status( $status = 'enabled' ) {
 
 			$checkout_type = give_stripe_get_checkout_type();
 
 			if ( 'redirect' === $checkout_type ) {
-				 return 'give-hidden';
+				return 'give-hidden';
 			}
 
 			return '';
 		}
 
 		/**
-		 * Connect button to connect with Stripe account.
+		 * Stripe Account Manager Field.
 		 *
-		 * @param string $value        Actual value.
-		 * @param string $option_value Option value.
+		 * This function will add a custom admin settings field to display
+		 * Stripe Account Management field where all the connected Stripe
+		 * accounts will display irrespective of the Stripe account connected
+		 * via Connect method or Manual method.
+		 *
+		 * @param array  $field        Field Arguments.
+		 * @param string $option_value Field Value.
+		 *
+		 * @return mixed|void
+		 * @since 2.7.0
 		 *
-		 * @since  2.5.0
-		 * @access public
 		 */
-		public function stripe_connect_field( $value, $option_value ) {
+		public function stripe_account_manager_field( $field, $option_value ) {
+
+			$stripe_accounts = give_stripe_get_all_accounts();
+			$default_account = '';
+
+			// Set account as default.
+			if ( is_array( $stripe_accounts ) && count( $stripe_accounts ) === 1 ) {
+				$stripe_account_keys = array_keys( $stripe_accounts );
+				$default_account     = $stripe_account_keys[0];
+			} else {
+				$default_account = give_stripe_get_default_account_slug();
+			}
+
+			$site_url            = get_site_url();
+			$modal_title         = __( '<strong>You are connected! Now this is important: Please configure your Stripe webhook to finalize the setup.</strong>', 'give' );
+			$modal_first_detail  = sprintf(
+				'%1$s %2$s',
+				__( 'In order for Stripe to function properly, you must add a new Stripe webhook endpoint. To do this please visit the <a href=\'https://dashboard.stripe.com/webhooks\' target=\'_blank\'>Webhooks Section of your Stripe Dashboard</a> and click the <strong>Add endpoint</strong> button and paste the following URL:', 'give' ),
+				"<strong>{$site_url}?give-listener=stripe</strong>"
+			);
+			$modal_second_detail = __( 'Stripe webhooks are required so GiveWP can communicate properly with the payment gateway to confirm payment completion, renewals, and more.', 'give' );
```

Comment:
Because translated strings are user-defined, the translated string should be escaped.